api/db/migrations/20190626102247_remove_organizationRoleId_column.js

```diff
@@ -0,0 +1,30 @@
+const TABLE_NAME = 'memberships';
+const TABLE_NAME_COPY = 'memberships2';
+
+exports.up = async function(knex) {
+
+  await knex.schema.createTable(TABLE_NAME_COPY, (table) => {
+    table.increments('id').primary();
+    table.bigInteger('userId').references('users.id').index();
+    table.bigInteger('organizationId').references('organizations.id').index();
+    table.unique(['userId', 'organizationId']);
+  });
+
+  const rows = await knex.select('id', 'userId', 'organizationId').from(TABLE_NAME);
+
+  await knex.batchInsert(TABLE_NAME_COPY, rows);
+
+  await knex.schema.dropTable(TABLE_NAME);
+  await knex.schema.renameTable(TABLE_NAME_COPY, TABLE_NAME);
+
+  await knex.schema.table(TABLE_NAME, (table) => {
+    table.string('organizationRole').defaultTo('OWNER');
```

Comment:
je pense qu'il faut qu'on réfléchisse à la valeur par défaut. Si la valeur par défaut c'est `OWNER`, ça veut dire qu'il peut y en avoir plusieurs dans la même orga, or il y a une règle métier qui dit que ce n'est pas possible...

Reply:
Je pense qu'on peut aussi en profiter pour marquer cette colonne `not nullable`.

Reply:
Done.
J'ai fixé la valeur par défaut : MEMBER